li-utils/src/main/pegasus/com/linkedin/common/MLFeatureSetUrn.pdl

```diff
@@ -0,0 +1,28 @@
+namespace com.linkedin.common
+
+/**
+ * Standardized MLFeatureSet identifier.
+ */
+@java.class = "com.linkedin.common.urn.MLFeatureSetUrn"
+@validate.`com.linkedin.common.validator.TypedUrnValidator` = {
+  "accessible" : true,
+  "owningTeam" : "urn:li:internalTeam:datahub",
+  "entityType" : "mlFeatureSet",
+  "constructable" : true,
+  "namespace" : "li",
+  "name" : "MlFeatureSet",
+  "doc" : "Standardized feature set identifier.",
+  "owners" : [ "urn:li:corpuser:fbar", "urn:li:corpuser:bfoo" ],
+  "fields" : [ {
+    "name" : "platform",
+    "type" : "com.linkedin.common.urn.DataPlatformUrn",
```

Comment:
cc @hsheth2 I am concerned about using data platform urn here tho it is already being referenced by the MLModel. Are we planning to just add platforms for everything as entities?

Reply:
Data platform already seems to have the precedent of adding a bunch of vaguely related items - it has a type parameter so I think it might just be fine to continue with this sort of approach

https://github.com/linkedin/datahub/blob/master/gms/impl/src/main/resources/DataPlatformInfo.json